presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -102,8 +102,66 @@ Property Name                                      Description
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
+================================================== ============================================================ ==========
+
+Amazon S3 Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Hive connector also allows querying data stored in Amazon S3. The SQL support 
+for S3 tables is the same as for HDFS tables.
+
+To access tables stored in S3, you must specify the AWS credential properties 
+``hive.s3.aws-access-key`` and ``hive.s3.aws-secret-key``. Instead, you can use
+``hive.s3.use-instance-credentials`` which if set to true, enables retrieving
+instance profile temporary AWS credentials.
+See http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/java-dg-roles.html
+
+Configuration Properties
+------------------------
+
+================================================== ============================================================ ==========
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ==========
+``hive.s3.aws-access-key``	                       AWS Access key.
```

Comment:
Indentation for the content in the "Description" column looks off. For this row and for a bunch of the other rows lower.